JustSaying/AwsTools/MessageHandling/SqsNotificationListener.cs

```diff
@@ -86,22 +85,16 @@ public SqsNotificationListener WithMessageProcessingStrategy(IMessageProcessingS
             _handlerMap.Add(typeof(T), handlerFunc);
         }
 
-        public void Listen()
+        public void Listen(CancellationToken cancellationToken)
```

Comment:
```suggestion
        public void Listen(CancellationToken cancellationToken = default)
```

Reply:
Does this work? I've not tried it before

Reply:
I mean the GitHub suggestions thing, you do three backticks and "suggestion" followed by the new code